templates/themes/maison/pagination/content_pagination.html.twig

```diff
@@ -0,0 +1,17 @@
+<div class="content-pagination bg-blue-light">
+    <div class="container content-pagination__container">
+        {% if previous is defined and previous is not null %}
+            <a href="{{ previous }}" class="content-pagination__link">
+                <i class="material-icons content-pagination__icon content-pagination__icon--prev">keyboard_backspace</i>
+                Previous Idea
+            </a>
+        {% endif %}
+        <a href="/inspirations" class="content-pagination__link">Back to Inspirations</a>
```

Comment:
don't we have something which generates those routes by e.g. Location RemoteId?

Reply:
This is only temporary approach. This url will be taken from the current inspiration menu position.